README.md

```diff
@@ -65,10 +65,16 @@ your crates, read [Building Crates so they Look Like C ABI Libraries][dev.to].
   to install `cargo-c` and do `cargo cinstall --prefix=/usr
   --destdir=/tmp/some-place` or something along those lines.
 
+If you plan to keep the bindings as a separate crate and do not need to autogenerate the headers you may just [populate Cargo.toml][diff-5]:
```

Comment:
If you need an example of same-crate-but-not-autogenerated-header then you could link to ebur128